### PR TITLE
add infinity water cell

### PR DESCRIPTION
**Pull request by Quetz4l**
infinity water cell from this PR https://github.com/GTNewHorizons/AE2FluidCraft-Rework/pull/189

changed the amount of liquid from `Integer.MAX_VALUE`  to `Integer.MAX_VALUE * 1_000_000L`  for creative cells 

default recipe (I'll make a different recipe for GTNH):
image

### CODE REVIEW

**Comment by boubou19** [comment]
i have nothing against this but why would someone uses this over a water hatch?

**Comment by Quetz4l** [comment]
Water hatches are used for crafting on the go (when one craft is always being made) or a water hatch + input fluid bus / TFFT + water hatch for obtaining water for further orders through the ME system.

I currently have a very high water consumption at the UXV tier, so I have to keep the TFFT just for water, as I have all other fluids on disks. I want to get rid of the TFFT and the complicated methods of obtaining water since there are no proper ways to do it.

**Comment by glowredman** [comment]
> i have nothing against this but why would someone uses this over a water hatch?

This is also nice in a non-GT context

**Comment by MCTBL** [comment]
https://github.com/GTNewHorizons/AE2FluidCraft-Rework/assets/30978504/07e8ebbc-e6d9-4574-99c9-d82bac701291

**Comment by MCTBL** [comment]
when there has other water cell in network, crafting job will can not start.

**Review by OneEyeMaker — CHANGES_REQUESTED** [review]

File: src/main/java/com/glodblock/github/common/storage/CreativeFluidCellInventory.java [hunk 1]
```diff
@@ -61,7 +61,7 @@ protected void loadCellFluids() {
                 if (fs == null) continue;
                 IAEFluidStack iaeFluidStack = Util.FluidUtil.createAEFluidStack(fs);
                 if (this.cellFluids.findPrecise(iaeFluidStack) == null) {
-                    iaeFluidStack.setStackSize(Integer.MAX_VALUE);
+                    iaeFluidStack.setStackSize(Long.MAX_VALUE);
```
[the remark this answers is not in the document]
Comment (Quetz4l) [reply src/main/java/com/glodblock/github/common/storage/CreativeFluidCellInventory.java:64]: The generation per second is sufficient, but not enough to have Integer.MAX_VALUE of water in the ME system. Large orders require much more water

[the remark this answers is not in the document]
Comment (OneEyeMaker) [reply src/main/java/com/glodblock/github/common/storage/CreativeFluidCellInventory.java:64]: If so, there should be another PR (probably for both AE2 and AE2FC mods) that addresses the issue of handling infinite storage. And only after such PR(s) this change can be applied without problems.

[the remark this answers is not in the document]
Comment (MCTBL) [reply src/main/java/com/glodblock/github/common/storage/CreativeFluidCellInventory.java:64]: ae is not good at handling infinite storage for now😂

[the remark this answers is not in the document]
Comment (Quetz4l) [reply src/main/java/com/glodblock/github/common/storage/CreativeFluidCellInventory.java]: Returned to `Integer.MAX_VALUE`, need to increase the volume of infinite cells in a separate pr

**Review by OneEyeMaker — APPROVED** [review]

**Pull request merged**